examples/frontend/pages/_app.tsx

```diff
@@ -4,11 +4,10 @@ import { ThemeProvider } from 'theme-ui'
 import { theme } from '../theme'
 import { RecoilRoot } from 'recoil'
 import { NearHooksProvider } from '@cura/hooks'
-import '@cura/components/dist/assets/fonts/index.css'
+import '@cura/components/assets/fonts/index.css'
```

Comment:
Previously, it would install `@cura/components` from npm, and all the packages files were under `/dist` folder and I couldn't export `css` from index without complicating things.

Now frontend is using the code directly from the monorepo without installing it from npm,

And because of this below, the imports from `@cura/components` are imported directly from `packages/components/src`
https://github.com/curaOS/source/blob/902883c9f5f350b41a2d23d00aa6b4b2f0d4ef44/examples/frontend/tsconfig.json#L17-L22